src/directives/jf_codemirror/jf_codemirror.js

```diff
@@ -39,10 +40,15 @@ class jfCodeController {
     $onInit() {
 	    this._formatModel();
 	    this.autofocus = this.autofocus === 'true';
+        let readOnlyParam = !this.allowEdit;
+
+        if(!_.isUndefined(this.noCursor) && this.noCursor) {
```

Comment:
Another option, to your consideration, without the use of readOnlyParam:
`const noCursor = !_.isUndefined(this.noCursor) && this.noCursor && 'nocursor';`
And then set the prop readOnly:
`readOnly: !this.allowEdit || noCursor `;